baseimages/terragrunt.hcl

```diff
@@ -0,0 +1,18 @@
+locals {
```

Comment:
I like the code simplicity, but we have agreed as a team that we don't want to introduce terragrunt (right now)